### ISSUE TITLE
Image paste does not working in all cases

### ISSUE THREAD

**Issue by blippercop** [glpi-project/glpi#12849]
### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### Is there an existing issue for this?

- [X] I have searched the existing issues

### Version

10.0.3

### Bug description

The image pase in the rich text editor does not work in all cases.

### Relevant log output

```shell
None
```

### Steps To reproduce

1. Create Word document with text and a screenshot like this:
2. 
<img width="522" alt="image" src="https://user-images.githubusercontent.com/96876285/193450202-4ddf5097-43c9-4967-a80f-879d93668d27.png">
3. Copy CTRL + A & CRTL + C
4. Paste it in GLPI (KB or Ticket)
5. See this:
6. 
<img width="519" alt="image" src="https://user-images.githubusercontent.com/96876285/193450239-dfce9e0d-42b6-4610-b2e7-460e64a8c2cc.png">

However on https://www.tiny.cloud/ it works. So must be something from GLPI?

### Your GLPI setup information

Plain

**Comment by cedric-anne** [comment]
Hi,

Do you have an error in the browser console log, or in GLPI server logs ?

**Comment by cedric-anne** [comment]
> However on https://www.tiny.cloud/ it works.

On my side, I have the same issue on https://www.tiny.cloud/ (at least on the demo editor from the home page). I tested on Google Chrome on Ubuntu.

**Comment by cconard96** [comment]
https://github.com/tinymce/tinymce/issues/7872

```
Not all images are supported; in particular word art and similar diagrams drawn in MS Word simply aren't available on the clipboard in any form. This can also extend to regular images depending on the context (there are a complex set of rules that determine what MS Word places on the clipboard).

The only browser which can import those correctly is Safari on macOS because it reads the file:// data (which is a rendering of the MS Word image) and makes that available to the editor during paste operations.
```

There is apparently a premium `PowerPaste` plugin that adds better handling. I don't know what it would take to try to recreate that functionality with a custom plugin. My initial testing showed 3 data sources in the clipboard during the TinyMCE `insertClipboardContent` function call (text/plain, text/html, and text/rtf). The RTF content isn't very readable to me, but I think this data source has the binary image content in hexadecimal.

**Comment by cedric-anne** [comment]
Hi,

Should be fixed by #12862.
